examples/some-verifier/frontend/src/components/Issuer.tsx

```diff
@@ -273,9 +274,10 @@ async function requestCredential(
   // eslint-disable-next-line no-constant-condition
   while (true) {
     try {
-      const itemSummary = await api
-        .getGrpcClient()
-        .waitForTransactionFinalization(txHash!);
+      const client = new ConcordiumGRPCClient(api.grpcTransport);
```

Comment:
This loop is still constructing a client per iteration.

Reply:
Well it appears there were 2 instances of this, so it's not the same one as the one referenced by your previous comment 😛